### PR TITLE
Enhancement/core quick fixes

### PR DESCRIPTION
**Pull request by ConnorBryan**
These are a series of quick fixes as discussed in the #prv-ui channel on Friday, January 18th.

### CODE REVIEW

File: src/molecules/ActionPanel/ActionPanel.tsx [hunk 1]
```diff
@@ -16,45 +12,33 @@ const ActionPanelBody = styled.div`
...
-    color: ${props => props.theme.linkHover};
-  }
 ` as StyledComponentClass<
-  DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>,
+  DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
   Theme
 >;
 
-ActionPanelLink.defaultProps = { as: 'a' };
+ActionPanelFooter.defaultProps = { as: 'div' };
```
Comment (nickmccurdy) [inline src/molecules/ActionPanel/ActionPanel.tsx]: This should be footer

File: src/molecules/StackedCard/StackedCard.tsx [hunk 2]
```diff
@@ -35,6 +35,10 @@ const StackedCardHeading = styled(Typography)`
   font-size: ${scale(1)};
 `;
 
+StackedCardHeading.defaultProps = {
+  as: 'div',
```
Comment (nickmccurdy) [inline src/molecules/StackedCard/StackedCard.tsx]: Should be header

File: src/molecules/ActionPanel/ActionPanel.tsx (hunk 1 above)
Comment (ConnorBryan) [reply src/molecules/ActionPanel/ActionPanel.tsx]: Oh, I thought <footer /> was for the document -- will fix.

File: src/molecules/StackedCard/StackedCard.tsx (hunk 2 above)
Comment (ConnorBryan) [reply src/molecules/StackedCard/StackedCard.tsx]: See above.

**Pull request merged**